### ISSUE TITLE
Page hide property for tags not working after update

### ISSUE THREAD

**Issue by gioacchinopoletto** [squidfunk/mkdocs-material#6157]
### Bug description

After upgrading to 9.4.4 (prior using 9.4.2) the `hide` property on a page not working, and tags start appearing on page top.
Rollback to 9.4.2 all work fine.

### Related links

- [Hide Tags on page](https://squidfunk.github.io/mkdocs-material/setup/setting-up-tags/?h=hide#hiding-tags-on-a-page)

### Reproduction

[Archivio.zip](https://github.com/squidfunk/mkdocs-material/files/12840431/Archivio.zip)

### Steps to reproduce

1. Add `tags` on `plugin` section of `mkdocs.yml`
2. Add `hide` property for `tags` as described on documentation
4. Upgrade to 9.4.4
5. See that tags still appear on top of page (not occur prior updating from 9.4.2)

### Browser

Chrome, Safari, Firefox

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by squidfunk** [comment]
Thanks for reporting. However, this is not a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) – the `mkdocs.yml` alone contains 160 lines and there are 30-40 pages. Please create a minimal reproduction, and we're happy to look into it.

**Comment by gioacchinopoletto** [comment]
> Thanks for reporting. However, this is not a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) – the `mkdocs.yml` alone contains 160 lines and there are 30-40 pages. Please create a minimal reproduction, and we're happy to look into it.
> 
> Also please make sure to use the [info plugin](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/#creating-a-zip-file) (**mandatory**).

I have try to generate a minimal reproduction with info plugin but I receive these errors on console:

```
gpoletto@Yoda mkdocs-minimal % mkdocs build
INFO    -  Started archive creation for bug report

Please name your bug report (2-4 words): 01
Traceback (most recent call last):                     
  File "/usr/local/bin/mkdocs", line 8, in <module>                                     
    sys.exit(cli())                                            
             ^^^^^                                                  
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1157, in __call__                                                 
    return self.main(*args, **kwargs)    
           ^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                                  
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1078, in main        
    rv = self.invoke(ctx)   
         ^^^^^^^^^^^^^^^^                                                                       
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)                                    
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                       
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)                                            
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/mkdocs/__main__.py", line 286, in build_command
    build.build(cfg, dirty=not clean)                                                                   
  File "/usr/local/lib/python3.11/site-packages/mkdocs/commands/build.py", line 277, in build
    config = config.plugins.on_config(config)                                                 
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                       
  File "/usr/local/lib/python3.11/site-packages/mkdocs/plugins.py", line 527, in on_config
    return self.run_event('config', config)                                                      
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                        
  File "/usr/local/lib/python3.11/site-packages/mkdocs/plugins.py", line 507, in run_event
    result = method(item, **kwargs)                                                             
             ^^^^^^^^^^^^^^^^^^^^^^                                                            
  File "/usr/local/lib/python3.11/site-packages/material/plugins/info/plugin.py", line 130, in on_config
    "\n".join(sorted([                                                                       
                     ^                           
  File "/usr/local/lib/python3.11/site-packages/material/plugins/info/plugin.py", line 131, in <listcomp>
    "==".join([package.name, package.version])                                            
TypeError: sequence item 0: expected str instance, NoneType found                                                                    
gpoletto@Yoda mkdocs-minimal % 
```

**Comment by gioacchinopoletto** [comment]
I will attach the minimal package created that have same error (2 pages: index, tags) and config file.
[mkdocs-minimal.zip](https://github.com/squidfunk/mkdocs-material/files/12843484/mkdocs-minimal.zip)

**Comment by squidfunk** [comment]
Thanks for providing the minimal reproduction and for checking which version worked last, that made fixing this problem very straight-forward! We refactored tags layout in 9.4.3, which caused this problem. Fixed in 53749f2ac.

**Comment by squidfunk** [comment]
Released as part of [9.4.5](https://squidfunk.github.io/mkdocs-material/changelog/#9.4.5).
